src/views/Home/components/EarnAssetCard.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Heading, Card, CardBody, ArrowForwardIcon } from '@pancakeswap-libs/uikit'
+import { NavLink } from 'react-router-dom'
+import pools from 'config/constants/pools'
+
+const StyledFarmStakingCard = styled(Card)`
+  background: linear-gradient(#53dee9, #7645d9);
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 344px;
+  width: 100%;
+  ${({ theme }) => theme.mediaQueries.lg} {
+    margin: 0;
+    max-width: none;
+  }
+`
+const Label = styled(Heading)`
+  color: ${({ theme }) => theme.colors.display};
+`
+const CardMidContent = styled(Heading).attrs({ size: 'xl' })`
+  color: white;
+  font-size: 35px;
+`
+const Row = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  color: #1fc7d4, 100%;
+`
+
+const EarnAssetCard = () => {
+  const TOKENS_TO_EXCLUDE = ['SYRUP']
+  const tokenNames = pools.reduce((accum, pool) => {
+    if (TOKENS_TO_EXCLUDE.includes(pool.tokenName)) {
+      return accum
+    }
+
+    return [...accum.slice(0, 3), pool.tokenName]
```

Comment:
Why don't you just put the first 3 tokens in the exclude list?

Reply:
This line returns the first 4 tokens from the list of tokens. The card is meant to display 4 tokens according to the Figma file:
![figma-assetsCard](https://user-images.githubusercontent.com/10050194/104604974-37fb0d80-567e-11eb-992c-b7951eb68c57.png)